docs/content/docs/dev/table/sqlClient.md

```diff
@@ -812,4 +812,35 @@ Flink SQL> RESET pipeline.name;
 
 If the option `pipeline.name` is not specified, SQL Client will generate a default name for the submitted job, e.g. `insert-into_<sink_table_name>` for `INSERT INTO` statements.
 
+### Monitoring job status
```

Comment:
Please also modify the expression about the management of the job lifecycle here.
![image](https://user-images.githubusercontent.com/33114724/218389489-6b17e64b-0f52-4e53-b84b-2ab5e06ca7af.png)